frontend/src/components/transactions/ReceiptRow.tsx

```diff
@@ -1,18 +1,24 @@
+import BN from "bn.js";
 import React from "react";
 import { Row, Col } from "react-bootstrap";
 
-import * as T from "../../libraries/explorer-wamp/transactions";
+import ActionRow from "./ActionRow";
 
+import * as T from "../../libraries/explorer-wamp/transactions";
+import Gas from "../utils/Gas";
+import Balance from "../utils/Balance";
+import { truncateAccountId } from "../../libraries/formatting";
 import { displayArgs } from "./ActionMessage";
 
 export interface Props {
-  receipt: T.ReceiptOutcome;
+  receipt: T.ExecutionOutcomeReceipts;
 }
 
 class ReceiptRow extends React.Component<Props> {
-  render() {
-    const { receipt } = this.props;
-
+  renderRow = (
+    receipt: T.ExecutionOutcomeReceipts,
+    convertedReceipt = true
+  ) => {
```

Comment:
There is no need to make it a method. Just put the implementation right in the `render` function and replace `this.renderRow()` with `<ReceiptRow receipt={executedReceipt} />`.